### ISSUE TITLE
Loss  value is abnormal

### ISSUE THREAD

**Issue by DanpengDing** [ellisdg/3DUnetCNN#340]
During training 3dircadb dataset（CT）, the loss value is very abnormal. My label is [0,1]
this is  data
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/b8c115f4-4b1c-4e53-82ec-837d41b930c8)
this is the label
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/d637f2d8-8d5a-46c3-9f84-146e87933202)
this is the loss value during traing
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/a7faa31e-07c6-4a3e-877e-44ca39cb22cb)

i only change the config
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/7bc8c802-c2bb-46ca-8633-0457a1cbe075)
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/3f1ccc27-6773-4932-ad96-f12937e5cc47)
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/9e098cbe-aee0-40b7-a750-706312e698ba)

What should I do with this problem

white is lable ,green is prediction   bad
![image](https://github.com/ellisdg/3DUnetCNN/assets/93312605/125e28b0-c3a6-4047-a1f9-6f47268e5c94)

**Comment by ellisdg** [comment]
How many epochs have you run the training for? The loss value of the first epoch are not indicative of how much the model is learning. It often takes hundreds of epochs for the loss value to plateau.

**Comment by ellisdg** [comment]
Ok. I see your configuration posted [here](https://github.com/ellisdg/3DUnetCNN/issues/249#issuecomment-2105537049).

Based on the prediction in green you provided, it doesn't look random. That means the model is learning something. It is just not learning as much as we would like.

This is just my intuition, but I think the model is having a hard time with the task because the task as presented is too hard. Just looking at the images you posted the contrast for the objects you are trying to segment is very low. Since you are using CT data, I would window the data to the intensity range that gives you high contrast for the task you want the model to perform. 
See this link for a description of windowing: [https://radiopaedia.org/articles/windowing-ct](https://radiopaedia.org/articles/windowing-ct)

You could also try to optimize model training for it to perform better. You could look at different loss functions, different models, etc.

I should note, though, that you have a small dataset, so it is unlikely that you will see incredible results even after optimization.
